Userland/Utilities/js.cpp

```diff
@@ -1041,6 +1048,16 @@ void ReplObject::initialize_global_object()
     define_native_function("load", load_file, 1, attr);
     define_native_function("save", save_to_file, 1, attr);
     define_native_function("loadJSON", load_json, 1, attr);
+
+    define_native_accessor(
+        "_",
+        [](JS::VM&, JS::GlobalObject&) {
+            return last_value.value();
+        },
+        [](JS::VM& vm, JS::GlobalObject& global_object) {
+            return vm.throw_completion<JS::ReferenceError>(global_object, JS::ErrorType::DescWriteNonWritable, "_");
```

Comment:
This is ok for now, but in the old implementation _ sort of served as a proxy: once you'd assign something to it all magic is lost, it becomes a proper variable. Only when nothing was found for the lookup it'd return the last value. 

Node does it like this:

```
> "foo"
'foo'
> _
'foo'
> _ = 42
Expression assignment to _ now disabled.
42
> _
42
> "bar"
'bar'
> _
42
> 
```

TODO is fine :^)